diff --git a/node_modules/@agoric/web-components/src/wallet-connection/watchWallet.js b/node_modules/@agoric/web-components/src/wallet-connection/watchWallet.js
index 27fd83d..5e6a0c0 100644
--- a/node_modules/@agoric/web-components/src/wallet-connection/watchWallet.js
+++ b/node_modules/@agoric/web-components/src/wallet-connection/watchWallet.js
@@ -52,10 +52,11 @@ export const watchWallet = async (chainStorageWatcher, address) => {
       chainStorageWatcher.watchLatest(
         ['data', `published.wallet.${address}.current`],
         value => {
+          // Don't hang on empty data.
+          res();
           const { offerToPublicSubscriberPaths: currentPaths } = value;
           if (currentPaths === lastPaths) return;
 
-          res();
           publicSubscriberPathsNotifierKit.updater.updateState(
             harden(currentPaths),
           );
